apps/api_web/lib/api_web/views/stop_view.ex

```diff
@@ -23,6 +23,12 @@ defmodule ApiWeb.StopView do
     field: :parent_station
   )
 
+  has_one(
+    :zone,
+    type: :zone,
+    field: :zone
```

Comment:
Actually, can you avoid the extra function definition if you make this `zone_id`?